omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
@guzzilar The complexity has been reduced.

From https://github.com/omise/omise-prestashop/pull/19#discussion_r97259710

> The knowledge of how to charge has been packed in `Charge` class.

> The controller class `OmisePaymentModuleFrontController` is control the flow.

The change below is out of scope of this pull request but it can make more understanding.

https://github.com/omise/omise-prestashop/blob/ba59c62cd78411fb07655e3dc5932f9daef50e51/omise/controllers/front/payment.php